src/main/java/bio/terra/common/sam/exception/SamConnectionException.java

```diff
@@ -0,0 +1,16 @@
+package bio.terra.common.sam.exception;
+
+import bio.terra.common.exception.BadRequestException;
+
+/**
+ * This exception is thrown when a request fails due to a connection issue, most often a timeout.
+ */
+public class SamConnectionException extends BadRequestException {
```

Comment:
that doesn't seem like it's a bad request? Probably a service unavailable extension?

Reply:
Whoops, that's what I get for copying other exceptions. Changed to generic internal server exception